wavefront_lambda/__init__.py

```diff
@@ -63,7 +75,17 @@ def wavefront_wrapper(*args, **kwargs):
         if not server:
             raise ValueError("Environment variable WAVEFRONT_URL is not set.")
         auth_token = os.environ.get('WAVEFRONT_API_TOKEN')
-        if not auth_token:
+        # WAVEFRONT_PROXY != 0 use WavefrontProxyReporter. default WavefrontDirectReporter
+        proxy = False
+        try:
+            int(os.getenv('WAVEFRONT_PROXY', "0"))
+        except ValueError:
+            pass
+        finally:
+            proxy = bool(proxy)
+        # WAVEFRONT_PORT PORT default 2878
+        port = os.environ.get("WAVEFRONT_PORT", 2878)
```

Comment:
We need to get this port , only if proxy = true. Something like below should work.
declare auth_token
if proxy : 
    get WAVEFRONT_PORT  //retrieve env variable
else :
   auth_token = get WAVEFRONT_API_TOKEN  //retrieve env variable
   if not auth_token :
           throw error

Reply:
I agree to get port when querying environment with default value this lazy.
It's not as clear as explicitly declare the variable. and logic is more clear with if proxy else check token throw. 
updated code per suggestion.